apps/coscrad-frontend/src/components/nav-menu/nav-menu.spec.tsx

```diff
@@ -13,7 +13,7 @@ describe('NavBar', () => {
     it('should render successfully', () => {
         const { baseElement } = renderWithProviders(
             <MemoryRouter>
-                <NavBar />
+                <NavMenu />
```

Comment:
Good improvement in naming. This is more declarative!

Reply:
Thumbs up